docs/docs-beta/docs/getting-started/quickstart.md

```diff
@@ -0,0 +1,177 @@
+---
+title: "Dagster quickstart"
+description: Learn how to quickly get up and running with Dagster
+sidebar_position: 30
+sidebar_label: "Quickstart"
+---
+
+# Build your first Dagster project
+
+Welcome to Dagster! In this guide, you'll use Dagster to create a basic Extract, Transform, Load (ETL) pipeline that extracts data from a CSV file and transforms it.
+
+## What you'll learn
+
+- How to set up a basic Dagster project
+- How to create a Dagster asset for each step of the ETL process
+- How to use Dagster's built-in features to monitor and execute your pipeline
+
+## Prerequisites
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+- Basic Python knowledge
+- Python 3.8+ installed on your system. Refer to the [Installation guide](/getting-started/installation) for information.
+</details>
+
+## Step 1: Set up the Dagster environment
+
+1. Open the terminal and create a new directory for your project:
+
+   ```bash
+   mkdir dagster-quickstart
+   cd dagster-quickstart
+   ```
+
+2. Create and activate a virtual environment:
+
+   <Tabs>
+   <TabItem value="macos" label="MacOS">
+   ```bash
+   python -m venv venv
+   source venv/bin/activate
+   ```
+   </TabItem>
+   <TabItem value="windows" label="Windows">
+   ```bash
+   python -m venv venv
+   source venv\Scripts\activate
+   ```
+   </TabItem>
+   </Tabs>
+
+3. Install Dagster and the required dependencies:
+
+   ```bash
+   pip install dagster dagster-webserver pandas
+   ```
+
+## Step 2: Create the Dagster project structure
+
+:::info
+The project structure in this guide is simplified to allow you to get started quickly. As a next step, check out the [ETL pipeline tutorial](/tutorial/tutorial-etl) to learn how to build more complex pipelines using best practices.
```

Comment:
Do we call out project scaffolding in tutorial-etl?

If not, it would be good to indicate that we suggest scafffolding project structure with `dagster project scaffold`, and was not done for this quick start.

Reply:
That tutorial isn't written yet - another thing that was in here when I started editing. I think I'm gonna remove the tutorial mention and just add in a line about the scaffolding, since the tutorial isn't slated for the community beta.